### ISSUE TITLE
Use in particular app only

### ISSUE THREAD

**Issue by Anshulgada** [espanso/espanso#1445]
I wanted to make few keywords which work in particular apps only. I tried reading the app-specific config but coudn't really understand much. So please could someone tell me this.

**Comment by Anshulgada** [comment]
I tried and now it works on VS Code but not with pycharm. The title is different for every project, class shows empty and only exec is the possible way but it shows some error I don't understand. I'll attach screenshots below.

![image](https://user-images.githubusercontent.com/80207612/201664118-0a165b29-7b90-4440-b294-8740a367a237.png)
![image](https://user-images.githubusercontent.com/80207612/201664020-349511f5-5418-4bde-ac59-8d247d2f8a8d.png)

**Comment by kpym** [comment]
Things to know : 
- `filter_exec` is a regular expression and in regular expressions `\` has a special meanin. 
- In yaml, depending on how you enquote strings, `\` may have also special meaning (check "Flow Scalars" section in in [yaml-multiline.info](https://yaml-multiline.info/)).

So for example to put a windows path like `C:\Path\file.exe` you have three choices:
- `filter_exec: C:\\Path\\file.exe` (without quotes, escape `\` only as regular expression)
- `filter_exec: 'C:\\Path\\file.exe'` (with sigle quotes, escape `\` only as regular expression)
- `filter_exec: "C:\\\\Path\\\\file.exe"` (with double quotes, , escape `\` as regular expression and as yaml double quoted string) 

In conclusion, it is probaby simpler to use something like `filter_exec: pycharm` and not the full path.

**Comment by smeech** [comment]
I'll close this for now, as I suspect you've either worked it out or abandoned Espanso by now!
If you think the documentation could be improved, please let me know.
